db/seeds.rb

```diff
@@ -116,79 +119,96 @@
 )
 
 # projects
-timmys_vaccines_nigeria = Project.create!(
-                  title: "Timmy's vaccine shots",
+20.times do |n|
+  project1 = Project.create!(
+                  title: "Timmy's vaccine shots_#{n}",
                   price: 50000,
                   description: "These are malaria shots for little Timmy." * 3,
                   retired: false,
                   categories: [people_category],
-                  tenant_id: bridge_builders1.id
+                  tenant_id: Tenant.first.id
+                  )
+  project1.photos << Photo.create!(
+                  image: File.new("#{Rails.root}/app/assets/images/timmys_vaccines.jpg")
```

Comment:
Line is too long. [88/80]<br>Indent the first parameter one step more than the previous line.